src/Ladder.java

```diff
@@ -0,0 +1,33 @@
+import java.io.*;
+import java.util.HashMap;
+
+public class Ladder {
+
+    private HashMap<Integer,Integer> ladders;
+
+    public Ladder(int numLadders, BufferedReader bufferedReader)throws IOException{
+        ladders = new HashMap<>();
+
+        for(int i = 0 ; i < numLadders ; i++){
+
+            String inp[] = bufferedReader.readLine().trim().split(" ");
```

Comment:
should not take user input in Ladder class. Should be done in MainDriver.

Reply:
Will address this to this in the revision.